frontend/src/router/index.js

```diff
@@ -5,9 +5,18 @@ import ApplicantRegister from "@/views/ApplicantRegister"
 import ApplicationRegister from "@/views/ApplicationRegister"
 import Login from "@/views/Login"
 import PasswordFind from "@/views/PasswordFind"
+import MyApplications from "@/views/MyApplications"
 
 Vue.use(VueRouter)
 
+const requireAuth = () => (to, from, next) => {
+  if (this.$store.state.token.value !== "") {
+    return next
```

Comment:
```suggestion
    return next()
```
`next`는 함수라 호출해야되네요.